libcli.h

```diff
@@ -78,6 +91,13 @@ struct cli_filter {
   struct cli_filter *next;
 };
 
+enum command_types {
+  CLI_ANY_COMMAND,
+  CLI_REGULAR_COMMAND,
+  CLI_FILTER_COMMAND,
+  CLI_BUILDMODE_COMMAND
```

Comment:
A trailing `,` here will make clang-format format this as multiple lines (like you have here).

Reply:
Added.